### ISSUE TITLE
Few days events..

### ISSUE THREAD

**Issue by niedved** [robmonie/jquery-week-calendar#5]
I want to add an event witch start for ex. at 10am on Monday and End 3pm at Friday.. is it possible ?

**Comment by zenangst** [comment]
+1 - I would like to have this feature as well!

**Comment by Ases** [comment]
+1

**Comment by northtouch** [comment]
Hi, I've got the same problem.
Is anyone have an answer ???

sorry for my english, I'm french.

**Comment by Zambros** [comment]
+1 
Now to fix this I create N event for every day. :-\

**Comment by northtouch** [comment]
Yes, it's an idea. But the trouble is that you can think have lot of events instead of one !!!

**Comment by Zambros** [comment]
I use a workaround.
http://www.magicbussrl.it/
I use a join table to duplicate only date and not every fields.

Details

http://img267.imageshack.us/img267/9650/detailsm.jpg

List
http://img408.imageshack.us/img408/5466/listmb.jpg
